instance/models/openedx_instance.py

```diff
@@ -310,8 +310,23 @@ def _spawn_appserver(self, deployment_id=None):
         elif self.storage_type == self.S3_STORAGE:
             self.logger.info('Provisioning S3 bucket...')
             self.provision_s3()
-        self.logger.info('Provisioning RabbitMQ vhost...')
-        self.provision_rabbitmq()
+
+        # Provision redis server if stated explicitly or no app servers were created before (ie. this is the first one)
+        if self.cache_db == OpenEdXDatabaseMixin.REDIS or self.appserver_set.count() == 0:
```

Comment:
@swalladge please see the related conversation: https://chat.opencraft.com/opencraft/pl/ppf7ycskapnjfc9g861eeoqfxe